monai/transforms/spatial/array.py

```diff
@@ -483,13 +486,23 @@ def __init__(
             recompute_affine: whether to recompute affine based on the output shape. The affine computed
                 analytically does not reflect the potential quantization errors in terms of the output shape.
                 Set this flag to True to recompute the output affine based on the actual pixdim. Default to ``False``.
+            min_pixdim: minimally allowed input spacing. If provided, input image with a larger spacing than this value
+                will be kept the same (not be resampled to `pixdim`) by this transform, default to `None`.
+            max_pixdim: maximally allowed input spacing. If provided, input image with a smaller spacing than this value
+                will be kept the same (not be resampled to `pixdim`) by this transform, default to `None`.
 
         """
         self.pixdim = np.array(ensure_tuple(pixdim), dtype=np.float64)
+        self.min_pixdim = np.array(ensure_tuple(min_pixdim), dtype=np.float64)
```

Comment:
this will be a numpy array of `np.nan`:

```py
>>> from monai.utils import ensure_tuple
>>> import numpy as np
>>> np.array(ensure_tuple(None), dtype=np.float64)
array([nan])
```